rdf-star-cg-spec.html

```diff
@@ -1041,9 +1065,32 @@ <h2>SA-mode and PG-mode</h2>
       </ul>
     </section>
 
+    <section>
+      <h2>The seminal example</h2>
+
+      <p>The motivating example in the original RDF* paper [[RDF-STAR-FOUNDATION]] was on a provenance use-case, and is repeated below.</p>
+
+      <pre data-transform="updateExample"
+        data-content-type="text/x-turtle-star"
+        class="nohighlight example"
+      >
+        <!--
+        # the controversial seminal example
+        :bob foaf:name "Bob".
+        <<:bob foaf:age 23>> dct:creator <http://example.com/crawlers#c1> ;
+                             dct:source <http://example.net/listing.html> .
```

Comment:
I think the second triple should be written in its entirety so that readers don't think that the single occurrence of the *string* `<<:bob foaf:age 23>>` is a factor in determining that the two arcs discuss the same RDF* triple.

Reply:
@ericprud since this is a verbatim of the example in the original paper, I would rather keep it as is.